ha/core/action_handler/action_factory.py

```diff
@@ -41,9 +39,13 @@ def get_action_handler(event: HealthEvent, action: list) -> ActionHandler:
             ActionHandler: return Specific object of ActionHandler like NodeFruPSUActionHandler for node:fru:psu.
         """
         try:
-            if event.resource_type in EVENT_ACTION_HANDLERS_MAPPING:
+            if len(actions) == 0 or \
```

Comment:
If no actions are configured for this resource_type then we need to return base class. And base class act method should not do anything

Reply:
for DefaultActionHandler
if len(actions) it return with no work
but base class will return unimplemented so considering default here 

Here if no key for resource_type we not need any work